src/api/index.js

```diff
@@ -9,10 +9,11 @@ const API_ERROR_TOAST_ID = 'api-error-toast-id';
 const USER_LOCAL_KEY = 'user';
 const USER_MENTORSHIP_REQUEST = 'mentorship-request';
 
-const paths = {
+export const paths = {
```

Comment:
Is it in used outside of this file?

```suggestion
const paths = {
```

Reply:
Not yet. I thought it might be a good idea to use it in our integration (cy) tests. But I ran into an issue with cypres so I'll have to come back to it later at some point.